configs/FLYWOOH743PRO/config.h

```diff
@@ -0,0 +1,150 @@
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU       STM32H743
+
+#define BOARD_NAME          FLYWOOH743PRO
+#define MANUFACTURER_ID     FLWO
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_SDCARD
+
+#define BEEPER_PIN          PA15
+#define MOTOR1_PIN          PB0
+#define MOTOR2_PIN          PB1
+#define MOTOR3_PIN          PA0
+#define MOTOR4_PIN          PA1
+#define MOTOR5_PIN          PA2
+#define MOTOR6_PIN          PA3
+#define MOTOR7_PIN          PD12
+#define MOTOR8_PIN          PD13
+#define MOTOR9_PIN          PD14
+#define MOTOR10_PIN         PD15
+#define SERVO1_PIN          PE5
+#define SERVO2_PIN          PE6
+#define RX_PPM_PIN          PC7
+#define LED_STRIP_PIN       PA8
+#define UART1_TX_PIN        PA9
+#define UART2_TX_PIN        PD5
+#define UART3_TX_PIN        PD8
+#define UART4_TX_PIN        PB9
+#define UART6_TX_PIN        PC6
+#define UART7_TX_PIN        PE8
+#define UART8_TX_PIN        PE1
+#define UART1_RX_PIN        PA10
+#define UART2_RX_PIN        PD6
+#define UART3_RX_PIN        PD9
+#define UART4_RX_PIN        PB8
+#define UART6_RX_PIN        PC7
+#define UART7_RX_PIN        PE7
+#define UART8_RX_PIN        PE0
+#define I2C1_SCL_PIN        PB6
+#define I2C2_SCL_PIN        PB10
+#define I2C1_SDA_PIN        PB7
+#define I2C2_SDA_PIN        PB11
+#define LED0_PIN            PE3
+#define LED1_PIN            PE4
+#define SPI1_SCK_PIN        PA5
+#define SPI2_SCK_PIN        PB13
+#define SPI3_SCK_PIN        PB3
+#define SPI4_SCK_PIN        PE12
+#define SPI1_SDI_PIN        PA6
+#define SPI2_SDI_PIN        PB14
+#define SPI3_SDI_PIN        PB4
+#define SPI4_SDI_PIN        PE13
+#define SPI1_SDO_PIN        PD7
+#define SPI2_SDO_PIN        PB15
+#define SPI3_SDO_PIN        PB5
+#define SPI4_SDO_PIN        PE14
+#define ADC_VBAT_PIN        PC0
+#define ADC_RSSI_PIN        PC5
+#define ADC_CURR_PIN        PC1
+#define SDIO_CK_PIN         PC12
+#define SDIO_CMD_PIN        PD2
+#define SDIO_D0_PIN         PC8
+#define SDIO_D1_PIN         PC9
+#define SDIO_D2_PIN         PC10
+#define SDIO_D3_PIN         PC11
+#define PINIO1_PIN          PD10 //9V BEC
+#define PINIO2_PIN          PD11 //CAM1,2
+#define MAX7456_SPI_CS_PIN  PB12
+#define GYRO_1_EXTI_PIN     PB2
+#define GYRO_2_EXTI_PIN     PE15
+#define GYRO_1_CS_PIN       PC15
+#define GYRO_2_CS_PIN       PE11
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, MOTOR1_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 6, MOTOR7_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 8, MOTOR9_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 9, MOTOR10_PIN,   1,   0) \
```

Comment:
Are you sure this is correct? We only support 8 motors

Reply:
Sorted.

```
Currently active Timers:
-----------------------
TIM1:
    CH1 : LED_STRIP
TIM2: FREE
TIM3: FREE
TIM4:
    CH3 : SERVO 1
    CH4 : SERVO 2
TIM5: FREE
TIM6: FREE
TIM7: FREE
TIM8:
    CH2 : DSHOT_BITBANG 2
    CH3 : DSHOT_BITBANG 1
TIM12: FREE
TIM13: FREE
TIM14: FREE
TIM15:
    CH1 : SERVO 3
    CH2 : SERVO 4
TIM16: FREE
TIM17: FREE

Currently active DMA:
--------------------
DMA1 Stream 0: LED_STRIP
DMA1 Stream 1: DSHOT_BITBANG 2
DMA1 Stream 2: DSHOT_BITBANG 1
DMA1 Stream 3: SPI_SDO 1
DMA1 Stream 4: SPI_SDI 1
DMA1 Stream 5: SPI_SDO 4
DMA1 Stream 6: SPI_SDI 4
DMA1 Stream 7: FREE
DMA2 Stream 0: ADC 1
DMA2 Stream 1: ADC 3
DMA2 Stream 2: FREE
DMA2 Stream 3: FREE
DMA2 Stream 4: FREE
DMA2 Stream 5: FREE
DMA2 Stream 6: FREE
DMA2 Stream 7: FREE
```